tests/e2e/tests/plugins/VscodeKubernetesPlugin.spec.ts

```diff
@@ -38,19 +35,16 @@ const devfileUrl: string = 'https://raw.githubusercontent.com/eclipse/che/main/t
 const factoryUrl: string = `${TestConstants.TS_SELENIUM_BASE_URL}/f?url=${devfileUrl}`;
 const sampleName: string = 'nodejs-web-app';
 const subRootFolder: string = 'app';
-let workspaceName: string;
 
 suite(`The 'VscodeKubernetesPlugin' test`, async () => {
     suite('Create workspace', async () => {
         test('Create workspace using factory', async () => {
             await browserTabsUtil.navigateTo(factoryUrl);
         });
 
-        test('Wait until created workspace is started', async () => {
-            await ide.waitAndSwitchToIdeFrame();
-            workspaceName = await workspaceNameHandler.getNameFromUrl();
-            CheReporter.registerRunningWorkspace(workspaceName);
+        workspaceHandlingTests.obtainWorkspaceNameFromStartingPage();
```

Comment:
It's strange to see code between the tests.

IMHO, it's part of `Wait until created workspace is started` test.